server/services/utils/conditionsFormatter.js

```diff
@@ -0,0 +1,40 @@
+const moment = require('moment');
+const DATE_FIELD = 'appointmentDate';
+const ALWAYS_REQUIRED = ['additionalConditions', 'nomisId'];
+
+module.exports = {formatConditionsInput};
+
+function formatConditionsInput(inputObject, selectedConditionsConfig) {
```

Comment:
Now that this file doesn't do any validating I think this can be streamlined at some point.